tf/modules/oonidevops_github_user/main.tf

```diff
@@ -0,0 +1,189 @@
+        "logs:ListTagsLogGroup",
+        "logs:StartLiveTail",
+        "logs:StartQuery",
+        "logs:StopLiveTail",
+        "logs:StopQuery",
+        "logs:TestMetricFilter",
+        "iam:Generate*",
+        "iam:Get*",
+        "iam:List*",
+        "iam:Simulate*",
+        "rds:Describe*",
+        "rds:Download*",
+        "rds:List*",
+        "route53-recovery-cluster:Get*",
+        "route53-recovery-cluster:ListRoutingControls",
+        "route53-recovery-control-config:Describe*",
+        "route53-recovery-control-config:GetResourcePolicy",
+        "route53-recovery-control-config:List*",
+        "route53-recovery-readiness:Get*",
+        "route53-recovery-readiness:List*",
+        "route53:Get*",
+        "route53:List*",
+        "route53:Test*",
+        "route53domains:Check*",
+        "route53domains:Get*",
+        "route53domains:List*",
+        "route53domains:View*",
+        "route53resolver:Get*",
+        "route53resolver:List*",
+        "s3:DescribeJob",
+        "s3:Get*",
+        "s3:List*",
+        "secretsmanager:Describe*",
+        "secretsmanager:GetResourcePolicy",
+        "secretsmanager:List*",
+        "servicediscovery:DiscoverInstances",
+        "servicediscovery:DiscoverInstancesRevision",
+        "servicediscovery:Get*",
+        "servicediscovery:List*",
+        "ssm:Describe*",
+        "ssm:Get*",
+        "ssm:List*",
+        "states:Describe*",
+        "states:GetExecutionHistory",
+        "states:List*",
+        "vpc-lattice:GetAccessLogSubscription",
+        "vpc-lattice:GetAuthPolicy",
+        "vpc-lattice:GetListener",
+        "vpc-lattice:GetResourcePolicy",
+        "vpc-lattice:GetRule",
+        "vpc-lattice:GetService",
+        "vpc-lattice:GetServiceNetwork",
+        "vpc-lattice:GetServiceNetworkServiceAssociation",
+        "vpc-lattice:GetServiceNetworkVpcAssociation",
+        "vpc-lattice:GetTargetGroup",
+        "vpc-lattice:ListAccessLogSubscriptions",
+        "vpc-lattice:ListListeners",
+        "vpc-lattice:ListRules",
+        "vpc-lattice:ListServiceNetworks",
+        "vpc-lattice:ListServiceNetworkServiceAssociations",
+        "vpc-lattice:ListServiceNetworkVpcAssociations",
+        "vpc-lattice:ListServices",
+        "vpc-lattice:ListTagsForResource",
+        "vpc-lattice:ListTargetGroups",
+        "vpc-lattice:ListTargets"
+    ]
+    resources = ["*"]
+  }
+}
+
+resource "aws_iam_user_policy" "oonidevops_github" {
+  name = "OONIDevopsGithubPolicy"
+  user = aws_iam_user.oonidevops_github_user.name
+  policy = data.aws_iam_policy_document.oonidevops_github.json
+}
+
+resource "aws_iam_access_key" "oonidevops_github_user" {
+    user = aws_iam_user.oonidevops_github_user.name
+} 
+
+resource "aws_secretsmanager_secret" "oonidevops_github_user_secrets" {
+  name = "oonidevops/github_user/access_key_json"
+  
+  tags = var.tags
+}
+
+resource "aws_secretsmanager_secret_version" "oonidevops_github_user_secrets" {
```

Comment:
Small nitpick, it's not needed to specify the type of a label in terraform, since you always refer to the resource by it's full name which includes the type in it, `resource "some_type" "some_name"` becomes `some_type.some_name`, so there is no need to disambiguate.

You should drop the `_secrets` suffix from here since it's redundant.